api/src/controllers/application.js

```diff
@@ -199,6 +204,18 @@ router.post("/:id/notify/:template", passport.authenticate(["referent", "young"]
     } else if (template === SENDINBLUE_TEMPLATES.young.REFUSE_APPLICATION) {
       emailTo = [{ name: `${application.youngFirstName} ${application.youngLastName}`, email: application.youngEmail }];
       params = { ...params, message, cta: `${APP_URL}/mission` };
+    } else if (template === "new") {
+      // when it is a new application, there are 2 possibilities
+      if (mission.isMilitaryPreparation === "true") {
+        const referentManagerPhase2 = await getReferentManagerPhase2(application.youngDepartment);
+        emailTo = [{ name: `${referentManagerPhase2.firstName} ${referentManagerPhase2.lastName}`, email: referentManagerPhase2.email }];
```

Comment:
(au pif) Est-ce qu'on ferait pas quand même un fallback pour éviter un crash si on n'a rien trouvé comme référent manager de phase 2 et que du coup on ferait quand même au référent ? (c'est peut-être idiot, mais c'est juste que comme j'ai vu trois if dans `getReferentManagerPhase2` je me dit qu'on peut tomber dans un cas moisi, mais ma proposition est peut-être encore plus moisie).

```suggestion
        if (referentManagerPhase2) emailTo = [{ name: `${referentManagerPhase2.firstName} ${referentManagerPhase2.lastName}`, email: referentManagerPhase2.email }];
        else emailTo = [{ name: `${referent.firstName} ${referent.lastName}`, email: referent.email }];
```

Reply:
yup